lnbits/wallets/corelightning.py

```diff
@@ -29,9 +29,14 @@ class CoreLightningWallet(Wallet):
     __node_cls__ = CoreLightningNode
 
     def __init__(self):
-        self.rpc = settings.corelightning_rpc or settings.clightning_rpc
-        self.ln = LightningRpc(self.rpc)
+        rpc = settings.corelightning_rpc or settings.clightning_rpc
+        if not rpc:
+            raise ValueError(
+                "cannot initialize CoreLightningWallet: "
+                "missing corelightning_rpc or clightning_rpc"
```

Comment:
maybe omit `clightning_rpc` in this message as its just for backwards compat

Reply:
updated 901ece8d37297fb876800e5714ea123d197a0325